plugin.audio.kxmxpxtx.bandcamp/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.kxmxpxtx.bandcamp" name="Bandcamp" version="0.0.4.1+matrix.1" provider-name="kxmxpxtx">
```

Comment:
Your version scheme is a bit odd. Major.minor.revision is required

Reply:
Are you talking about "+matrix.1" part? Or about x.x.x.x?
Should I make it "0.4.1+matrix.1" instead?

Reply:
Yes please keep version as thre params: major minor revision. The local version identifier (+matrix.1) is fine

Reply:
done